go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -306,6 +308,20 @@ func (m *ModelClient) create(opModel *OperationModel) ([]ovsdb.Operation, error)
 			Until:   "!=",
 			Rows:    []ovsdb.Row{{"name": opModel.Name}},
 		})
+	} else if o[0].Table == "Logical_Router_Policy" {
+		lrp, ok := opModel.Model.(*nbdb.LogicalRouterPolicy)
```

Comment:
is this check needed? can there be a case where o[0] is "Logical_Router_Policy" but the Model is something different?

Reply:
Not really, added it to be on the safe side + using the `lrp` var later.